simplic-interval/Simplic.Interval/Simplic.Interval/Model/Interval.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simplic.Interval
+{
+    /// <summary>
+    /// Interval definition
+    /// </summary>
+    public class Interval
+    {
+        #region Fields
+
+        private Guid guid;
```

Comment:
we dont use backing fields. only properties. instead of `public Guid Guid { get => guid; set => guid = value; }` we just use `public Guid Guid { get; set; }`